front/app/utils/surveyUtils.ts

```diff
@@ -0,0 +1,22 @@
+import { Layout } from '@jsonforms/core';
```

Comment:
I would recommend using our own `PageCategorization` instead, since the jsonforms types are horrible. See `front/app/components/Form/typings.ts`

Reply:
Ah smart - didn't realize we had that type. Indeed - those types are really bad :sweat_smile: Thanks!